api_core/tests/unit/test_grpc_helpers.py

```diff
@@ -231,14 +237,15 @@ def test_callable_response(self):
         expected_request = operations_pb2.GetOperationRequest(name='meep')
         expected_response = operations_pb2.Operation(name='moop')
 
-        def on_get_operation(request):
-            return expected_response
+        on_get_operation = mock.Mock(
+            spec=('__call__'), return_value=expected_response)
```

Comment:
@jonparrott You aren't using a tuple there.

Also, FWIW `spec=('__call__',)` doesn't do anything (it **might** on a `MagicMock`, but not a `Mock`). All `Mock`-s are callable and you can't disable it.

Reply:
Yeah but it's more explicit, I like to say "I'm only using this mock for `__call__`"